HelpMyStreetFE/HelpMyStreetFE/Controllers/YotiController.cs

```diff
@@ -26,17 +26,18 @@ public YotiController(IOptions<YotiOptions> options, IValidationService validati
 
         public IActionResult Authenticate()
         {
-            var viewModel = new AuthenticateViewModel { ClientSdkId = _options.ClientSdkId, DomId = _options.DomId, ScenarioId = _options.ScenarioId };
+            var viewModel = new AuthenticateViewModel { UserId = HttpContext.User.FindFirst(ClaimTypes.NameIdentifier).Value, ClientSdkId = _options.ClientSdkId, DomId = _options.DomId, ScenarioId = _options.ScenarioId };
             return View(viewModel);
         }
 
-        [HttpPost]
-        public async Task<IActionResult> ValidateToken([FromBody] ValidateTokenRequest request, CancellationToken token = default)
+        [HttpGet]
+        public async Task<IActionResult> ValidateToken(string userId, string token, CancellationToken cancellationToken)
```

Comment:
We shouldnt need to keep passing up the userid?

Reply:
@gburnhill can clarify, but I think we have to pass it for the mobile journey.

Reply:
Unsurprisingly @Goolyio you were totally right about id